### PR TITLE
fix: button text overflows in menu

### PR DESCRIPTION
**Pull request by im-adithya**
### Describe the changes you have made in this PR

Added `overflow: hidden` to `menuItemButton` styling

### Type of change (Remove other not matching type)

- `fix`: Bug fix (non-breaking change which fixes an issue)

### Screenshots of the changes (If any)

**Before**
Screenshot from 2022-06-03 22-06-54

**After**
Screenshot from 2022-06-03 22-10-38

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

**Comment by escapedcat** [comment]
Instead of just cutting it off hard it could be nice to try using [overflow ellipsis](https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow). Could be combined with a default [`title`-attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title) which would show the full name on hover.

**Comment by im-adithya** [comment]
@escapedcat can you please check how the recent commit is working on your system. I'm facing a weird issue because of which the wallet icon is not being displayed before the account name (as you can see in the above screenshot) and is now (after the recent commit) of variable widths despite having the `w-6` style added.

Screenshot from 2022-06-04 13-13-01

(Mostly it should be an issue on my side with the CSS but it still was a bit weird before too, so I'm asking just to make sure everything's fine!)

> Could be combined with a default [title-attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title) which would show the full name on hover.
I was thinking of the same, would help in increasing accessibility. Will add it after resolving the above issue!

**Comment by escapedcat** [comment]
> I'm facing a weird issue because of which the wallet icon is not being displayed before the account name (as you can see in the above screenshot)

My understanding is that it was removed on purpose because it didn't provide any value to the user. See: #916

> > Could be combined with a default [title-attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title) which would show the full name on hover.
> 
> I was thinking of the same, it would help in increasing accessibility! Will add it after resolving the above issue!

👍

**Comment by im-adithya** [comment]
> See: https://github.com/getAlby/lightning-browser-extension/pull/916

Not the badge, I was talking about the wallet icon^

**Comment by escapedcat** [comment]
> Not the badge, I was talking about the wallet icon^

Oh, I see. Maybe @secondl1ght knows

**Comment by bumi** [comment]
I think the icon should be there. seems to be because of the length of the text?

**Comment by im-adithya** [comment]
Ah, must be it. Applying shrink-0 should fix it then!

**Comment by im-adithya** [comment]
Made all the necessary changes, ready for review!

Screenshot from 2022-06-04 19-59-29

File: src/app/components/AccountMenu/index.tsx [hunk 1]
```diff
@@ -83,9 +83,12 @@ function AccountMenu({ title, subtitle, showOptions = true }: Props) {
                   selectAccount(accountId);
                 }}
                 disabled={loading}
+                title={account.name}
               >
-                <WalletIcon className="w-6 h-6 -ml-0.5 mr-2 opacity-75 text-gray-700 dark:text-neutral-300" />
-                {account.name}&nbsp;
+                <WalletIcon className="w-6 h-6 -ml-0.5 mr-2 shrink-0 opacity-75 text-gray-700 dark:text-neutral-300" />
+                <div className="overflow-hidden text-ellipsis whitespace-nowrap">
```
Comment (escapedcat) [inline src/app/components/AccountMenu/index.tsx:89]: Could this be a `span`? Block element within this element seems not semantic.

**Review by escapedcat — CHANGES_REQUESTED** [review]

File: src/app/components/AccountMenu/index.tsx (hunk 1 above)
Comment (im-adithya) [reply src/app/components/AccountMenu/index.tsx]: Done!

**Review by escapedcat — APPROVED** [review]

**Review by bumi — APPROVED** [review]

**Pull request merged**